roles/zookeeper/molecule/default/verify.yml

```diff
@@ -3,17 +3,18 @@
   hosts: all
   gather_facts: false
   tasks:
-  - name: check zookeeper package status
-    package:
-      name: "{{ item }}"
-      state: present
-    check_mode: true
-    register: pkg_status
-    loop:
-      - zookeeperd
-      - zookeeper
+    - name: Check ruby package status
+      ansible.builtin.package:
+        name: "{{ item }}"
+        state: present
+      check_mode: true
+      register: pkg_status
+      loop:
+        - ruby-switch
+        - ruby2.6-dev
+        - ruby2.6
 
-  - name: test for zookeeper packages
-    assert:
-      that:
-        - not pkg_status.changed
+    - name: Test for ruby packages
```

Comment:
Here again, it seems odd that we are checking for ruby packages instead of checking that zookeeper is installed and running. Was this copy pasta?

Reply:
copy pasta from `roles/example`